docs/types/operators.rst

```diff
@@ -108,3 +108,68 @@ value it referred to previously.
             assert(y.length == 0);
         }
     }
+
+.. index:: precedence
```

Comment:
While at it, I'd improve the index entry. Let's make it the main entry for this topic and nest it under `operators`.

```suggestion
.. index:: ! operators; precedence
```